packages/clay-pagination-bar/src/PaginationBarWithBasicItems.tsx

```diff
@@ -152,6 +152,14 @@ export const ClayPaginationBarWithBasicItems: React.FunctionComponent<IProps> =
 		return item;
 	});
 
+	const totalPages = Math.ceil(totalItems / activeDelta);
+
+	React.useEffect(() => {
+		if (onPageChange && activePage > totalPages) {
+			onPageChange(1);
+		}
```

Comment:
@bryceosterhaus maybe not better to check this in the `onClick` of the delta above? so we don't need an extra `useEffect`.

Reply:
I went with the useEffect since its possible that the user could change the total items or active delta via the props and the same bug would result. By using `useEffect`, we cover all those cases as well as onClick.

Reply:
Yeah, great! I had forgotten that.